src/statefunctions/gamestates/convertedkan.cc

```diff
@@ -40,5 +40,5 @@ std::unique_ptr<GameState> ConvertedKan(std::unique_ptr<GameState> state) {
   return state;
 }
 
-REGISTER_ROUTE(ConcealedKan, StateFunctionType::kConcealedKan);
+REGISTER_ROUTE(ConvertedKan, StateFunctionType::kConvertedKan);
```

Comment:
this typo brings up an interesting point, should the registration process throw some sort of error if two things are trying to register to the same route?

Reply:
I think probably, but I want to revisit this later when start having dynamic routing for 3 player.